docs/python.md

```diff
@@ -0,0 +1,13 @@
+# Python
```

Comment:
are we planning to also have a `rust.API` doc? 

it feels that this section repeats what's in the Index.md

Reply:
you could add the link to https://docs.rs/cql2/latest/cql2/  in the menu 

so maybe have a API menu with `python` and `rust` submenu 🤷